src/post/post.service.ts

```diff
@@ -33,8 +33,12 @@ export class PostService {
     return `This action returns a #${id} post`;
   }
 
-  update(id: number, updatePostDto: UpdatePostDto) {
-    return `This action updates a #${id} post`;
+  async update(id: string, updatePostDto: UpdatePostDto){ 
+    const updatingPost = await this.postModel.findByIdAndUpdate(id, updatePostDto, { new: true });
```

Comment:
According to @vanglevan's feedback, you shouldn't send the DTO as an argument to update the Model. Instead, you shoould create a sample model, assign its properties based on the DTO, and then pass it to findByIdAndUpdate().
You can reference the code below: 
![image](https://github.com/NhatNguyen1502/open-universe-be/assets/133724931/72014964-ebe9-4ea7-bb1e-cd09e46c2d2c)